src/flavors/duwamish_flavor/config.yml

```diff
@@ -565,6 +618,302 @@ place:
+          optional: true
+        - name: difficulty_breathing
+          type: binary_toggle
+          prompt: _(I am having difficulty breathing)
+          display_prompt: _(I am having difficulty breathing:)
+          content: yes_no
+          annotation: _(Please be sure to seek medical attention if needed.)
+          optional: true
+        - name: headache
+          type: binary_toggle
+          prompt: _(I am experiencing a headache)
+          display_prompt: _(I am experiencing a headache:)
+          content: yes_no
+          annotation: _(Please be sure to seek medical attention if needed.)
+          optional: true
+        - name: other_physical_effects
+          type: text
+          prompt: _(Please note other physical effects you are experiencing)
+          display_prompt: _(Other physical effects I experienced:)
+          annotation: _(Please be sure to seek medical attention if needed.)
+          optional: true
+        - name: source  #internally used name for this form element
+          type: dropdown  #form type
+          prompt: _(Select one of the following to identify the source of the bad air quality.)  #display prompt
+          display_prompt: _(Pollution source observed:)  #text used when a submitted report is viewed later
+          content: air_quality_source  #points to dynamic_form_content section below, to avoid possibly duplicating complex form content
+          optional: false
+        - name: normal_activities
+          type: radio_big_buttons
+          prompt: "_(Please indicate your agreement with the following statement: My normal activities are affected by the air quality in my neighborhood.)"
+          display_prompt: _(My normal activities are affected by the air quality in my neighborhood:)
+          content: likert_agree_5
+          optional: false
+        - name: activities_impacted
+          type: dropdown
+          prompt: _(Select one of the following to indicate how your normal activities are impacted.)
+          display_prompt: _(My normal activities were impacted in the following way:)
+          content: activities_impacted
+          optional: false
+        - name: additional_comments
+          type: text
+          prompt: _(Provide any additional comments)
+          display_prompt: _(Additional comments:)
+          optional: true   
+    tree:
+      name: location_type
+      dataset: trees
+      datasetSlug: trees
+      icon_url: /static/css/images/markers/marker-tree.png
+      value: tree
+      label: _(Tree)
+      fields:
+        - name: title
+          type: text
+          prompt: _(Title)
+          placeholder: _(Enter title)
+          optional: false
+        - name: description
+          type: textarea
+          prompt: _(Description)
+          placeholder: _(Description...)
+          optional: false
+
+  # define shared form element content here
+  dynamic_form_content:
```

Comment:
Yeah, I can see the argument for folding the `dynamic_form_content` section back into the main part of the dynamic form config. The thought behind abstracting it out was to avoid duplicated content, but that use case hasn't really emerged yet, so I agree it's adding an extra layer of complexity as is.

I'll refactor to eliminate the extra section :)